ams/server/src/main/java/com/netease/arctic/server/table/ServerTableIdentifier.java

```diff
@@ -10,28 +11,32 @@ public class ServerTableIdentifier {
   private String catalog;
   private String database;
   private String tableName;
+  private TableFormat format;
```

Comment:
After adding the field `format` to `ServerTableIdentifier`, we can remove this field from `TableMetadata`.
```java
public class TableMetadata implements Serializable {
  ...

  private ServerTableIdentifier tableIdentifier;

  private TableFormat format;
  ...
}
```

Reply:
fixed.